pkg/gerr/error.go

```diff
@@ -12,8 +12,8 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
-	"openpitrix.io/openpitrix/pkg/logger"
-	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/notification/pkg/logger"
```

Comment:
change to openpitrix.io/logger

logger.NewLogger().WithDepth(5).Error  -> logger.New().WithDepth(2).Errorf

Reply:
openpitrix.io/logger does not contain this function:NewLogger().

Reply:
Pls change "logger.NewLogger().WithDepth(5).Error" to "logger.New().WithDepth(2).Errorf" .

Reply:
done